src/app/app.module.ts

```diff
@@ -1,24 +1,39 @@
 import { BrowserModule } from '@angular/platform-browser';
 import { NgModule } from '@angular/core';
-
+import { FormsModule } from '@angular/forms'
```

Comment:
This should only be necessary if you are using ngmodel, which I don't think we are.  If we aren't we should remove this.

Reply:
Gotcha, I use the Ng Model on my slider values which is how the stackblitz examples were structured